py_zipkin/zipkin.py

```diff
@@ -270,6 +274,15 @@ def start(self):
                     flags=existing_zipkin_attrs.flags,
                     is_sampled=existing_zipkin_attrs.is_sampled,
                 )
+            elif self.firehose_handler is not None:
+                # If it has gotten here, the only thing that is
+                # causing a trace is the firehose. So we force a trace
+                # with sample rate of 0
+                report_root_timestamp = True
+                self.zipkin_attrs = create_attrs_for_span(
+                    sample_rate=0.0,
```

Comment:
I'm a little confused here - wouldn't we want a 100% sampling rate on the span? Or is that not how it works?

Reply:
Sorry I forgot to respond to you. As far as the attrs is concerned, the sample rate is 0%. The firehose is separate from the sample rate.